PyPrograms/Raspberry Pi Detector/main.py

```diff
@@ -0,0 +1,12 @@
+# Raspberry Pi Detector
+# By @bupboi1337
+
+import os
+hostname = "raspberrypi.local" # Raspberry Pi Local Hostname
+response = os.system("ping -c 1 " + hostname) # Ping raspberrypi.local
+
+# check the response
+if response == 0: # If the ping went through correctly
+  print hostname, 'detected! There is a Raspberry Pi on your Network!'
```

Comment:
I should replace detected! with connection successful and add a space before these messages so they don't look wierd